mm2src/mm2_event_stream/src/lib.rs

```diff
@@ -34,6 +34,7 @@ pub struct EventStreamConfiguration {
     pub access_control_allow_origin: String,
     #[serde(default)]
     active_events: HashMap<String, EventConfig>,
+    pub worker_path: Option<String>,
```

Comment:
Can we make this `PathBuf` instead with the default value of `PathBuf::from("worker.js")`?

Reply:
It's not needed for native though, that's why I made it optional.

Reply:
We can use wasm target condition for this field

Reply:
I guess I can make it `PathBuf` with wasm cfg attribute.

Reply:
Done